spec/models/blood_sugar_spec.rb

```diff
@@ -10,4 +10,28 @@
     it { should belong_to(:patient).optional }
     it { should belong_to(:user).optional }
   end
+
+  describe '#diabetic?' do
+    [{ blood_sugar_type: :random, blood_sugar_value: 300 },
+     { blood_sugar_type: :fasting, blood_sugar_value: 200 },
+     { blood_sugar_type: :post_prandial, blood_sugar_value: 301 }].each do |row|
```

Comment:
How about `300`, to be consistent with the others?

Reply:
I thought it might better to have a slightly wider range to test, but it doesn't really matter, I'll update to the boundary values.